README.md

```diff
@@ -1,5 +1,13 @@
 # search-a-tags
 
+[![NPM Version](https://img.shields.io/npm/v/search-a-tags-hacktoberfest.svg)](https://www.npmjs.com/package/search-a-tags-hacktoberfest)
+[![NPM Downloads](https://img.shields.io/npm/dt/search-a-tags-hacktoberfest.svg)](https://www.npmjs.com/package/search-a-tags-hacktoberfest)
+[![Github Repo Size](https://img.shields.io/github/repo-size/AryanAg08/search-a-tags-hacktoberfest.svg)](https://github.com/AryanAg08/search-a-tags-hacktoberfest)
```

Comment:
Can you change all occurrences of `AryanAg08` to `arshadkazmi42` otherwise it will point to your forked repository. You can do a find and replace